front/app/components/admin/DateSinglePicker/index.tsx

```diff
@@ -15,6 +20,25 @@ const Container = styled.div`
   border-radius: ${(props) => props.theme.borderRadius};
   border: solid 1px ${colors.borderDark};
 
+  /*
```

Comment:
In the report this was called "the selected day that doesn't have enough contrast", even though it's not the selected day. But I thought just fixing this is faster (and better for users anyway) than explaining this.